CliWrap/StandardBufferHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace CliWrap
+{
+    /// <summary>
+    /// The standard implementation of <see cref="IStandardBufferHandler"/>
```

Comment:
Let's call it `the default implementation` to avoid using word "standard" in two different meanings.